vsix/download_artifact.ps1

```diff
@@ -0,0 +1,37 @@
+$apiUrl = 'https://ci.appveyor.com/api'
+$token = '<your-api-token>'
+$headers = @{
+  "Authorization" = "Bearer $token"
+  "Content-type" = "application/json"
+}
+$accountName = $env:APPVEYOR_ACCOUNT_NAME
+$projectSlug = $env:APPVEYOR_PROJECT_SLUG
+$buildId = $env:APPVEYOR_BUILD_ID
+
+$downloadLocation = 'C:\projects'
+
+# get project with last build details
+$project = Invoke-RestMethod -Method Get -Uri "$apiUrl/projects/$accountName/$projectSlug/builds/$buildId" -Headers $headers
+
+# we assume here that build has a single job
+# get this job id
+$jobId = $project.build.jobs[0].jobId
+
+# get job artifacts (just to see what we've got)
+$artifacts = Invoke-RestMethod -Method Get -Uri "$apiUrl/buildjobs/$jobId/artifacts" -Headers $headers
+
+# here we just take the first artifact, but you could specify its file name
+# $artifactFileName = 'Conan.VisualStudio\bin\Release\Conan.VisualStudio.vsix'
```

Comment:
leftover

Reply:
The way to go is to use the actual name of the artifact, but I need to learn how to save the artifact and give it a name (`/` doesn't work very well in names). This is something to fix in the near future.